deploy/dev/reload.sh

```diff
@@ -0,0 +1,36 @@
+Reload
+
+#!/bin/bash
+set -xeuo pipefail
+
+. utils.sh
+
+# Script for making it easy to make a change locally and redeploy
+cd ..
+./bin/build
+
+cd dev
```

Comment:
```suggestion
pushd ..
  ./bin/build
popd
```

2 changes:
1. use pushd
2. you had `cd dev` but you should have done `cd deploy` to be in the correct directory

Reply:
updated